android/src/com/mapswithme/maps/MwmActivity.java

```diff
@@ -2720,4 +2723,31 @@ public void onGlobalLayout()
                               : UiUtils.getStatusBarHeight(getApplicationContext()));
     }
   }
+
+  @Override                                                                                                                                        
+    public boolean onKeyUp(int keyCode, KeyEvent event) {                                                                                            
+      switch (keyCode) {                                                                                                                             
+        case KeyEvent.KEYCODE_DPAD_DOWN:                                                                                                             
+          Statistics.INSTANCE.trackEvent(Statistics.EventName.ZOOM_OUT);                                                                             
+          AlohaHelper.logClick(AlohaHelper.ZOOM_OUT);                                                                                                
```

Comment:
Remove  this code `AlohaHelper.logClick(AlohaHelper.ZOOM_OUT);` please,  all statistics will be sent within `Statistics.INSTANCE.trackEvent` . P.S. I know that you take existing code, but it's wrong.

Reply:
Done